doc/integrator/internationalization.rst

```diff
@@ -34,14 +34,20 @@ Build your application.
 
 The files to translate are:
 
-* ``demo/locale/<lang>/LC_MESSAGES/demo-client.po`` for ngeo client
-* ``demo/locale/<lang>/LC_MESSAGES/demo-server.po`` for the server part (should be empty for ngeo interfaces)
+* ``geoportal/<package>/locale/<lang>/LC_MESSAGES/demo-client.po`` for ngeo client
+* ``geoportal/<package>/locale/<lang>/LC_MESSAGES/demo-server.po`` for the server part (should be empty for ngeo interfaces)
 
 .. note::
 
    All the ``#, fuzzy`` strings should be verified and the line should be removed
    (if the line is not removed, the localisation will not be used).
 
+To update the po files you should run the specific target:
```

Comment:
Should we say something about which part is no longer automatic / when exactly it is necessary to run this target?

Reply:
| which part is no longer automatic

What do you mean here?

Reply:
If I understand correctly, these updates are no longer "automatic" in the sense that the GMF build will include this. Instead, the user must do a build with a specific target to rebuild the files when the texts have changed?

Reply:
Shouldn't we add something here?
https://github.com/camptocamp/c2cgeoportal/blob/2.3/geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl#L34

Reply:
@jwkaltz yes, this is not automatic. @sbrunner updated!

Reply:
Thanks :-)